docs/release_notes/release_notes.md

```diff
@@ -14,7 +14,52 @@ For the full release notes, see the [Changelog](https://github.com/Katello/katel
 
 ## Features
 
+### Scenario Installer
+
+The installer has been updated to use the foreman-installer with a new feature known as scenarios. Scenarios provide a way to capture a precise set of installation options that can be re-used to install anything from a bare Foreman, to Katello to a Capsule. The initial scenarios being released are: Foreman, Katello and Caspule. For more information and examples, see the installation instructions.
+
+### Host Unification
+
```

Comment:
Prior to Katello 2.0, Hosts were represented by two objects, Hosts and Content Hosts.  Under some cercumstances these two objects would be linked together but under other workflows they were not.  In addition only one Host could exist with a given hostname but many Content Hosts could be registered with the same hostname.  As part of Katello 3.0 we have attempted to unify hosts and content hosts into a single object.  The "Content Hosts" page still exists but eventually everything there would be integerated into a unified Hosts ui.

As a result of Host Unification there are many subtle changes:

* Registering a host using subscription-manager now creates a host object if one does not exist
* The "All Hosts" page and the "Content Hosts" page show the same list of host objects
* The "Content Hosts" page provides the same scoped search auto-completion as "All Hosts"
* Registering a host using subscription-manager will automatically reuse an existing host with the same hostname
* Content Host pages dealing with subscriptions and content actions will show an appropriate error if you visit them for a Host that had not been registered with subscription-manager
* Most 'hammer content-host' commands have moved to 'hammer host subscription' and 'hammer host content'